### DISCUSSION TITLE
How to add custom user registration

### DISCUSSION THREAD

**Discussion by joeynq** [logto-io/logto#2605] [Q&A] [answered]
I don't see document describe user sign up experiences. Where can I find it? Can we create it or customize it?

**Comment by gao-sun** [comment, marked as answer]
you can take a look at these articles. hope them helps

[The design considerations for a seamless sign-in experience (First Chapter)](https://silverhand.substack.com/p/the-design-considerations-for-a-seamless)
[The design considerations for a seamless sign-in experience (Second Chapter)](https://silverhand.substack.com/p/the-design-considerations-for-a-seamless-2d4)
